tests/services/text-to-id.test.js

```diff
@@ -0,0 +1,40 @@
+import textToId from '~/services/text-to-id'
+
+describe('#textToId', () => {
+  test('it should return an id text from a text', () => {
+    // given
+    const text = 'Yo, dev tes compétences au sein de Pix !'
+    const expectedId = 'yo--dev-te'
+
+    // when
+    const id = textToId(text)
+
+    // then
+    expect(id).toBe(expectedId)
+  })
+
+  test('when the text is small, it should return an id text from a text', () => {
```

Comment:
**question:** `when the text is small` => c'est la condition `text.length < 2` qui est testé? parce que `pix` fait plus de 2 caractères ^^